google-map-search.html

```diff
@@ -57,36 +63,128 @@
       },
 
       /**
-       * The search result.
+       * Latitude of the center of the search area.
+       * Ignored if `currentMapArea` is true.
        */
-      result: {
-        type: Object,
+      latitude: {
+        type: Number,
+        value: null
+      },
+
+      /**
+       * Longitude of the center of the search area.
+       * Ignored if `currentMapArea` is true.
+       */
+      longitude: {
+        type: Number,
+        value: null
+      },
+
+      /**
+       * Search radius, in meters.
+       * If `latitude` and `longitude` are not specified,
+       * the center of the currently visible map area is used.
+       *
+       * Ignored if `currentMapArea` is true.
+       *
+       * @attribute radius
+       * @type number
+       * @default null
+       */
+      radius: {
+        type: Number,
+        value: null
+      },
+
+      /**
+       * Restrict the search only to the current map area.
+       *
+       * @attribute currentMapArea
+       */
+      currentMapArea: {
+        type: Boolean,
+        value: true
```

Comment:
for booleans, it's better to have the default value be false. When the attribute is applied, it becomes thruthy.